app/src/main/java/com/example/homework3/database/GithubCashedUsersDao.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.homework3.database
+
+import androidx.room.*
+import com.example.homework3.model.GithubCashedUser
+import com.example.homework3.retrofit.Item
+
+@Dao
+interface GithubCashedUsersDao {
+    @Query("SELECT * FROM githubuser")
+    suspend fun getAll(): List<Item.GithubUser>
+
+    @Query("SELECT * FROM githubuser WHERE id IN (:usersIds)")
+    suspend fun loadAllByIds(usersIds: IntArray): List<Item.GithubUser>
```

Comment:
тут тоже самое, из бд получаем просто юзера, когда доходит до отображения на списке, то оборачиваем в Item
```suggestion
    suspend fun loadAllByIds(usersIds: IntArray): List<GithubUser>
```